ansible/deploy_netvault.yml

```diff
@@ -2,24 +2,58 @@
   hosts: pg_standby
   become: true
   tasks:
+
     - name: "Download Netvault"
       get_url:
         url: "{{ netvault_url }}"
         dest: /tmp/netvault.tar.gz
+
     - name: "Unarchive Netvault"
       unarchive:
         src: /tmp/netvault.tar.gz
         dest: /tmp/
         remote_src: yes
+
     - name: "Create symlink for lib64"
       file:
         src: /usr/lib
         dest: /usr/lib64
         state: link
+
     # FIXME: deal with prompt for options
     - name: "Runs install script"
       #command: /tmp/netvault/install
       command: /bin/true
       when: false
+
+    - name: "Grants PostgreSQL access to netvault user"
+      blockinfile:
+        path: "{{ postgresql_config_home }}/pg_hba.conf"
```

Comment:
any reason not to add this to our pg_hba.conf template

Reply:
i thought about it. but since i am running this using extra_vars didn't want to mess with that template.  i'll improve this once we get confirmation that it is an accepted solution.